task_02/index.html

```diff
@@ -1,11 +1,180 @@
+              voluptas nulla pariatur?"
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="item" id="item3">
+        <div class="section-wrapper">
+          <div class="text-wrapper">
+            <div class="post-title">This is a title</div>
+            <div class="post-text">
+              "Sed ut perspiciatis unde omnis iste natus error sit voluptatem
+              accusantium doloremque laudantium, totam rem aperiam, eaque ipsa
+              quae ab illo inventore veritatis et quasi architecto beatae
+              vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia
+              voluptas sit aspernatur aut odit aut fugit, sed quia
+              consequuntur magni dolores eos qui ratione voluptatem sequi
+              nesciunt. Neque porro quisquam est, qui dolorem ipsum quia dolor
+              sit amet, consectetur, adipisci velit, sed quia non numquam eius
+              modi tempora incidunt ut labore et dolore magnam aliquam quaerat
+              voluptatem. Ut enim ad minima veniam, quis nostrum
+              exercitationem ullam corporis suscipit laboriosam, nisi ut
+              aliquid ex ea commodi consequatur? Quis autem vel eum iure
+              reprehenderit qui in ea voluptate velit esse quam nihil
+              molestiae consequatur, vel illum qui dolorem eum fugiat quo
+              voluptas nulla pariatur?"
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="item" id="item4">
+        <div class="section-wrapper">
+          <div class="text-wrapper">
+            <div class="post-title">This is a title</div>
+            <div class="post-text">
+              "Sed ut perspiciatis unde omnis iste natus error sit voluptatem
+              accusantium doloremque laudantium, totam rem aperiam, eaque ipsa
+              quae ab illo inventore veritatis et quasi architecto beatae
+              vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia
+              voluptas sit aspernatur aut odit aut fugit, sed quia
+              consequuntur magni dolores eos qui ratione voluptatem sequi
+              nesciunt. Neque porro quisquam est, qui dolorem ipsum quia dolor
+              sit amet, consectetur, adipisci velit, sed quia non numquam eius
+              modi tempora incidunt ut labore et dolore magnam aliquam quaerat
+              voluptatem. Ut enim ad minima veniam, quis nostrum
+              exercitationem ullam corporis suscipit laboriosam, nisi ut
+              aliquid ex ea commodi consequatur? Quis autem vel eum iure
+              reprehenderit qui in ea voluptate velit esse quam nihil
+              molestiae consequatur, vel illum qui dolorem eum fugiat quo
+              voluptas nulla pariatur?"
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+  </div>
 </body>
+<script>
+  function moveText() {
+    const section = document.querySelectorAll(".section-wrapper");
```

Comment:
It's not, since it's confusing and can't tell to what it refers. It will be changed to contentWrapper. The type of it is an object which holds section-wrapper (also will be changed to content-wrapper).